src/app/globals.css

```
@@ -44,6 +44,8 @@
 }
 
 :root {
+  --font-geist-sans: "Segoe UI", "Helvetica Neue", Arial, sans-serif;
```

Stylelint (17.6.0)

[error] Expected "Arial" to be "arial" (value-keyword-case)

(value-keyword-case)